scripts/configure_pyproject_toml.py

```diff
@@ -18,14 +18,24 @@
 import os
 from pathlib import Path
 
-import toml
+from importlib import import_module
+from importlib.util import find_spec
 from backend_support import backend, device_name
 
 path_to_version = Path("pennylane_lightning").absolute() / "core" / "_version.py"
 with open(path_to_version, encoding="utf-8") as f:
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
 
+has_toml = False
+toml_libs = ["tomllib", "tomli", "tomlkit"]
```

Comment:
Note that we should test (locally is fine) that the syntax works against all of these different libraries.

Reply:
Good suggestion! I re-use the logic in PL here but worth testing the use cases for all different toml libs.

Reply:
We need both `tomli` and `tomli-w` as `tomli` is read-only and `tomli-w` is required in `configure_pyproject_toml.py` to be able to update the setup config for other Lightning devices. I drop the `tomli` support as to keep the logic simple and straightforward.